pandas/tests/io/json/test_pandas.py

```diff
@@ -1675,7 +1675,7 @@ def test_to_json_indent(self, indent):
         "primaryKey":[
             "index"
         ],
-        "pandas_version":"0.20.0"
+        "pandas_version":"1.4.0"
```

Comment:
so its ok to change this as long as you add *another* schema that reads correctly that is defined in 0.20.0